pyproject.toml

```diff
@@ -27,27 +27,28 @@ classifiers = [
     "Programming Language :: Python :: 3",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
     "Topic :: Software Development",
     "Topic :: Scientific/Engineering :: Artificial Intelligence",
     "Topic :: Scientific/Engineering :: Information Analysis",
     "Topic :: Scientific/Engineering :: Image Recognition"
 ]
 dependencies = [
     "autogluon.common >=1, <2",
-    "aws-cdk-lib >=2.0.0,<2.70.0",
-    "aws-cdk.aws-batch-alpha >=2.0.0a1,<2.70.0a0",
-    "awscliv2 >=2.2.0,<2.3.0",
-    "boto3 >=1.26.0,<1.28.0",
-    "constructs >=10.0.0,<10.1.289",
-    "fsspec >=2023.5.0,<=2023.6.0",
-    "matplotlib >=3.4,<3.8",
-    "pandas >=2.0.0,<2.2.0",
+    "aws-cdk-lib >=2.0.0,<3",
+    "aws-cdk.aws-batch-alpha >=2.0.0a1,<2.96.0a0",
```

Comment:
Their change may be breaking even with micro version changes. Maybe we can keep the upper bound unchanged, and I'll work on a full update on aws-cdk (which involves a lot more stack code changes) after this PR.

Reply:
Reverted